esphome/components/ds2484/ds2484.cpp

```diff
@@ -0,0 +1,209 @@
+#include "ds2484.h"
+
+namespace esphome {
+namespace ds2484 {
+static const char *const TAG = "ds2484.onewire";
+
+void DS2484OneWireBus::setup() {
+  ESP_LOGCONFIG(TAG, "Setting up DS2484 1-wire bus...");
+  this->reset_device();
+  this->search();
+}
+
+void DS2484OneWireBus::dump_config() {
+  ESP_LOGCONFIG(TAG, "DS2484 1-wire bus:");
```

Comment:
[Do not use the component name in log messages](https://developers.esphome.io/architecture/logging/#minimizing-impact) -- that's what `TAG` is for. 🙂
```suggestion
  ESP_LOGCONFIG(TAG, "1-wire bus:");
```